qdrant/v1.2.x/configuration.md

```diff
@@ -207,7 +207,7 @@ tls:
 
 ## Authentication
 
-*Available since v1.2.0*
+*Available as of v1.2.0*
```

Comment:
@davidmyriel Could you open a PR to update all these occurrences throughout the documentation? There are a lot more.